File: Descending Order.py
# https://www.codewars.com/kata/descending-order/train/python

def Descending_Order(num):
    a = [i for i in str(num)]
    a.sort(reverse=True)
    a = int("".join(a))
    return a


def Descending_Order_V2(num):
    return int("".join(sorted(str(num), reverse=True)))


print(Descending_Order_V2(21445))

File: DescendingOrder.js
// https://www.codewars.com/kata/descending-order/train/javascript

function descendingOrder(num){
    let a = []
    num = num.toString()
    for (let i = 0; i < num.length; i++){
        a.push(num[i])
    }
    a = ((a.sort()).reverse()).join("")
    return parseInt(a)

}

function descendingOrderV2(n){
    return parseInt(String(n).split('').sort().reverse().join(''))
}

console.log(descendingOrder(21445))
